oap-server/server-starter/src/main/resources/ui-initialized-templates/mysql/mysql-root.json

```diff
@@ -0,0 +1,63 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+[
+   {
+      "id":"b6e3763c-882d-417f-8556-3cf0187cb8db",
+      "configuration":{
+         "children":[
+            {
+               "x":0,
+               "y":3,
+               "w":24,
+               "h":43,
+               "i":"0",
+               "type":"Widget",
+               "graph":{
+                  "type":"ServiceList",
+                  "dashboardName":"MySQL-Service",
+                  "fontSize":12,
+                  "showXAxis":false,
+                  "showYAxis":false,
+                  "showGroup":true
+               }
+            },
+            {
+               "x":0,
+               "y":0,
+               "w":24,
+               "h":3,
+               "i":"1",
+               "type":"Text",
+               "graph":{
+                  "fontColor":"blue",
+                  "backgroundColor":"white",
+                  "content":"Provide MySQL monitoring.",
```

Comment:
```suggestion
                  "content":"Provide MySQL Server monitoring through OpenTelemetry's MySQL Receiver",
```

Reply:
I use prometheus/mysqld_exporter and prometheus receiver in OpenTelemetry collector. Maybe not the OpenTelemetry's MySQL Receiver?

Reply:
OK, just make this accurate.